adapters/IronSource/IronSourceAdapter/GADMAdapterIronSourceBase.m

```diff
@@ -15,16 +15,18 @@
 #import "GADMAdapterIronSourceBase.h"
 
 @implementation GADMAdapterIronSourceBase {
-  BOOL _initSucceeded;
+  
 }
 
 // IronSource internal reporting const.
-NSString *const kGADMMediationName = @"AdMob";
+NSString *const kGADMAdapterIronSourceMediationName = @"AdMob";
+
 // IronSource parameters keys.
 NSString *const kGADMAdapterIronSourceAppKey = @"appKey";
 NSString *const kGADMAdapterIronSourceIsTestEnabled = @"isTestEnabled";
 NSString *const kGADMAdapterIronSourceInstanceId = @"instanceId";
 
+
 #pragma mark - Admob
 
 + (NSString *)adapterVersion {
```

Comment:
We should change this version to hardcode @"6.7.7.0" in here. We don't want to pull at runtime from IronSource's SDK, because we want to distinguish what version of the SDK and adapter the developer is running.

Hopefully they are only using 6.7.7.0 adapter with the 6.7.7 SDK, but we'd like to know for sure.

Reply:
Sure.